### DISCUSSION TITLE
Swagger supports the object externalDocs, but it's not rendered in the API Doc view

### DISCUSSION THREAD

**Discussion by VWACBrick** [Azure/api-management-developer-portal#2561] [Feature request]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

## Bug description

The OpenAPI specification includes an open for externalDocs object that would allow the user to include a URL that could be indicated on the API details view.

https://swagger.io/specification/v2/#externalDocumentationObject

{
  "description": "Read more information here",
  "url": "https://www.bing.com"
}

## Expected behavior

![image](https://user-images.githubusercontent.com/65634611/98410531-b22d6700-2029-11eb-9133-87f02c685366.png)

## Is your portal managed or self-hosted?

Self-hosted

## Release tag or commit SHA (if using self-hosted version)

2.5.1

## Additional context

There are other properties that can be set like TOU, Contact, and License, but this is already handled at a global level.

**Comment by azaslonov** [comment]
Hi @VWACBrick, this is not a developer portal issue specifically, the internal representation of API in APIM doesn't support this property. So for now, the only thing I can recommend is to file a request at aka.ms/apimwish.

**Comment by mikebudzynski** [comment]
@VWACBrick We have plans to improve OpenAPI support within the next several months. I'll keep this issue open for now. As a workaround, you can include a link in the API description (markdown syntax is supported).

**Comment by mikebudzynski** [comment]
Referencing #1456
